scrapy_zyte_api/handler.py

```diff
@@ -159,6 +191,19 @@ async def _download_request(
 
         return _process_response(api_response, request)
 
+    def _log_request(self, params):
+        if not self._must_log_request:
+            return
+        params = self._truncate_params(params)
+        logger.debug(f"Sending Zyte API extract request: {json.dumps(params)}")
```

Comment:
Should we also log the API endpoint that we're sending the request through? I.e. https://api.zyte.com/v1/extract.

My concern is that users might be interested if they're using a newer v2 or v3 endpoint later on. Or perhaps their own dedicated enterprise endpoint for that matter.

We could also choose to log this API endpoint once when the handler is created.

Reply:
> We could also choose to log this API endpoint once when the handler is created.

Yes, I think this is a good point but it may be best to delay it until we support alternative endpoints. And, even then, we might want to log it once at the beginning of the logs, to keep request-specific logs as short as possible, unless we support using multiple different endpoints on the same spider.